content/manuals/copilot/examples.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Example prompts for the Docker agent
+linkTitle: Example prompts
+description: |
+  Discover example prompts to interact with the Docker agent and learn how to
+  automate tasks like Dockerizing projects or opening pull requests.
+weight: 30
+---
+
+{{% restricted title="Early Access" %}}
+The Docker for GitHub Copilot extension is an [early access](/release-lifecycle#early-access-ea) product.
+{{% /restricted %}}
+
+Here are some examples of the types of questions you can ask the Docker agent:
+
+### Ask general Docker questions
+
+You can ask general question about Docker. For example:
+
+- `@docker what is a Dockerfile`
+- `@docker how do I build a Docker image`
+- `@docker how do I run a Docker container`
```

Comment:
should these have question marks? i tested them out as is and they seems to work, but adding the `?` would make it nicer to read imho given this is user facing